### ISSUE TITLE
turn off PDFBox logging

### ISSUE THREAD

**Issue by jeremybmerrill** [tabulapdf/tabula-java#16]
with `java.util.logging.Logger.getLogger('org.apache.pdfbox').setLevel(java.util.logging.Level::OFF)`'s Java equivalent.

perhaps leave it on with `--debug`?

**Comment by jeremybmerrill** [comment]
@jazzido Do you want to keep this on in all cases?

**Comment by jazzido** [comment]
No, but the log messages are useful for debugging `ObjectExtractor`. I'll add a switch to the cmd line to toggle them.

**Comment by jeremybmerrill** [comment]
Ah, super. I wasn't able to figure out how the `--debug` switch works (does it do anything?) so I turned it off entirely.
